src/api/routes/fee-rate.ts

```diff
@@ -0,0 +1,20 @@
+import * as express from 'express';
+import { addAsync, RouterWithAsync } from '@awaitjs/express';
+import { DataStore } from '../../datastore/common';
+import { FeeRate } from '@stacks/stacks-blockchain-api-types';
+
+export const FEE_RATE = 346;
```

Comment:
@asimm241 can you change this to `400` to match the current hard coded number in the wallets?